db/schema.rb

```diff
@@ -47,6 +47,7 @@
     t.text "description"
     t.time "event_time"
     t.date "event_date"
+    t.string "games"
```

Comment:
I am right to think that this should not be here? :thinking: If so ,then try doing the following to fix this:
1. `rails db:drop`
2. `rails db:create`
3. `rails db:migrate` 

And the commit and push the changes.